lib/model.js

```diff
@@ -1353,7 +1356,10 @@ Model.prototype.findAll = function(options) {
   tableNames[this.getTableName(options)] = true;
   options = optClone(options);
 
-  _.defaults(options, { hooks: true });
+  _.defaults(options, { hooks: true, kenophobic: false });
```

Comment:
Couldn't this be ` _.defaults(options, { hooks: true, kenophobic: this.options.kenophobic });`?